cmd/agent-js-worker/main.go

```diff
@@ -500,99 +526,320 @@ func createVDRs(resolvers []string, trustblocDomain, trustblocResolver string) (
 	return VDRs, nil
 }
 
-func agentOpts(opts *agentStartOpts) ([]aries.Option, error) {
+func agentOpts(startOpts *agentStartOpts) ([]aries.Option, error) {
 	msgHandler := msghandler.NewRegistrar()
 
-	var options []aries.Option
-	options = append(options, aries.WithMessageServiceProvider(msgHandler))
+	var ariesOptions []aries.Option
```

Comment:
not sure why you renamed them to 'ariesOpts', but they are not just aries opts any more. We renamed it anywhere to use the term `agent` instead of `aries`. They have trustbloc components too.

Reply:
I had renamed it to "ariesOpts" because I thought the call to `addStorageOptions(startOpts, options)` would look a bit confusing unless I somehow made it clear that it was for the agent... but you're right that it shouldn't be called `ariesOpts`. I renamed it back to `options`.